### PR TITLE
Added Control Center button to the startmenu

### PR DESCRIPTION
**Pull request by 1Spinne**
The title says everything

An icon is still missing
The gear is just a placeholder

### CODE REVIEW

File: src/app/desktop/desktop-startmenu/desktop-startmenu.component.html [hunk 1]
```diff
@@ -20,34 +20,38 @@
...
-    <input [(ngModel)]="searchTerm" aria-label="Search" class="search" placeholder="Search"/>
+    <input [(ngModel)]="searchTerm" aria-label="Search" class="search" placeholder="Search" />
   </div>
   <div class="buttons">
     <div (click)="logout()" class="button" id="logout">
-      <img alt="" class="icon" src="assets/desktop/img/logoutbutton.svg"/>
+      <img alt="" class="icon" src="assets/desktop/img/logoutbutton.svg" />
       Logout
     </div>
+    <div (click)="parent.hideStartMenu()" [queryParams]="{ device: parent.activeDevice.uuid }" class="button"
```
Comment (MarcelCoding) [inline src/app/desktop/desktop-startmenu/desktop-startmenu.component.html:46]: I think hiding the start menu isn't necessary.

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Review by MarcelCoding — DISMISSED** [review]

**Review by MarcelCoding — DISMISSED** [review]

File: src/app/desktop/desktop-startmenu/desktop-startmenu.component.scss [hunk 2]
```diff
@@ -185,6 +185,10 @@ $taskbar-height: 5.7%;
         height: auto;
         margin-right: 9px;
       }
+
+      .cc {
+        filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(200deg) brightness(101%) contrast(103%);
```
Comment (MarcelCoding) [inline src/app/desktop/desktop-startmenu/desktop-startmenu.component.scss:190]: Change the color in the svg file itself and don't use filters.

File: src/assets/desktop/img/controlcenter.svg [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (MarcelCoding) [inline src/assets/desktop/img/controlcenter.svg:1]: Minify svg, e.g. export using Inkscape as a "optimized svg"

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

File: src/app/desktop/desktop-startmenu/desktop-startmenu.component.scss (hunk 2 above)
Comment (1Spinne) [reply src/app/desktop/desktop-startmenu/desktop-startmenu.component.scss]: done

**Review by MarcelCoding — APPROVED** [review]

**Review by Tristan-H11 — APPROVED** [review]

**Pull request merged**